pkgs/top-level/haskell-packages.nix

```diff
@@ -56,11 +56,6 @@ rec {
       inherit crossSystem;
       selfPkgs = packages.ghcHEAD;
     };
-    ghcNokinds = callPackage ../development/compilers/ghc/nokinds.nix rec {
```

Comment:
There is another reference here in the same file, the `ghcNokinds` package set, which in turn references another obsolete file: `pkgs/development/haskell-modules/configuration-ghc-nokinds.nix`.